Services/UserService/Commands/CreateUserCommand.cs

```diff
@@ -0,0 +1,28 @@
+﻿using DTO;
+using UserService.Interfaces;
+using Kernel;
+
+public class CreateUserCommand : ICreateUser<string, string>
+
```

Comment:
Idk there're some strange empty lines in some files

Reply:
Wow, namespace isn't there. Thanks!